pkg/skaffold/schema/v1alpha2/config.go

```diff
@@ -128,15 +128,16 @@ type HelmDeploy struct {
 type KustomizeDeploy struct{}
 
 type HelmRelease struct {
-	Name           string                 `yaml:"name"`
-	ChartPath      string                 `yaml:"chartPath"`
-	ValuesFilePath string                 `yaml:"valuesFilePath"`
-	Values         map[string]string      `yaml:"values,omitempty"`
-	Namespace      string                 `yaml:"namespace"`
-	Version        string                 `yaml:"version"`
-	SetValues      map[string]string      `yaml:"setValues"`
-	Wait           bool                   `yaml:"wait"`
-	Overrides      map[string]interface{} `yaml:"overrides"`
+	Name            string                 `yaml:"name"`
+	ChartPath       string                 `yaml:"chartPath"`
+	ValuesFilePath  string                 `yaml:"valuesFilePath"`
+	ValuesFilePaths []string               `yaml:"valuesFilePaths"`
```

Comment:
It doesn't really make sense to support both ValuesFilePaths and ValuesFilePath to me, would you be up for making ValuesFilePath a []string instead of adding this second field?

We have a system for helping users make breaking changes to their config for things like this.

Reply:
Yes, I can do that. Should I create another schema? how does this system works?

Reply:
cc @nkubala can you help out here?

Reply:
@stevenbarragan the short version is that you need to git cp v1alpha2 to v1alpha3, and write a "transformer" to convert the yaml for users like this: https://github.com/GoogleContainerTools/skaffold/blob/master/pkg/skaffold/config/transform/transforms.go#L29

You can avoid making v1alpha3 the new default schema for now, in case we want to bundle in any other schema changes now.

Reply:
Ah sorry I somehow missed this. @stevenbarragan let me know if you need any more help here